translations/ui-users/en_US.json

```diff
@@ -396,7 +396,8 @@
     "accounts.actions.warning.item": "Item",
     "accounts.actions.warning.waiveAction": "waived",
     "accounts.error.exceeds": "Pay amount exceeds the selected amount",
-    "accounts.warning.itemSelected": "{total, number} items selected. {items, number} {items, plural, one {item} other {items}} cannot be {action} because they are already closed.",
+    "accounts.warning.itemSelected": "{total, number} {total, plural, one {item} other {items}} selected. {items, number} {items, plural, one {item} other {items}} cannot be {action} because {items, plural, one {it is} other {they are}} already closed.",
```

Comment:
To apply changes in en.json to other translations we need to run a Lokalize job. It's ok to change these files locally, but if you commit them they could be overridden by that job

That job is triggered when a translation key is changed, so it will automatically update
If you don't want to change the key - after merging a PR with changes in en.json please contact Peter Murray and ask to run a translations update job in that repository

Reply:
Hey @BogdanDenis, the two changes in en.json and en_US.json are same, so its okay if en_US.json gets overridden by the job. en_US.json is updated here just to make it work locally as it was not working with making translation update in en.json. So shall I proceed with the changes or revert en_US.json changes ?

Reply:
@manvendra-s-rathore it's better to revert changes to en_US.json and after merging this PR ask Peter Murray to run the translations update job.